biz.aQute.bndlib.tests/test/test/ProjectTest.java

```diff
@@ -1198,4 +1202,41 @@ public void testCopyRepo() throws Exception {
 		assertTrue(project.check());
 		assertTrue(ws.check());
 	}
+
+
+	@Test
+	public void testWarnOnDuplicateProperties(@InjectTemporaryDirectory
```

Comment:
I just noticed that there is already a field-level temp directory injected, you can use that rather than injecting a parameter as well.

Reply:
good catch. changed too.